pkg/webhook/parser.go

```diff
@@ -186,7 +186,7 @@ func (c Config) matchPolicy() *admissionreg.MatchPolicyType {
 	case strings.ToLower(string(admissionreg.Equivalent)):
 		matchPolicy = admissionreg.Equivalent
 	default:
-		matchPolicy = admissionreg.MatchPolicyType(c.MatchPolicy)
+		return nil
```

Comment:
Does the `Unrecognized values are passed through` comment on L180 still hold with this change? IIUC, this will ignore invalid values, which is different behavior from failurePolicy for example.

Reply:
Good point, fixed!

Reply:
@vincepri Are we okay with a different behavior than failurePolicy? i.e. failurePolicy uses any value you give and server returns error if invalid, but matchPolicy ignores invalid values.

Reply:
I think so, we probably want to avoid to set a value that's not supported